### PR TITLE
Bug 1212873 - Compute the 95% confidence interval for mean

### PR DESCRIPTION
**Pull request by stasm**
https://bugzilla.mozilla.org/show_bug.cgi?id=1212873

### CODE REVIEW

File: lib/phases/phase.js [hunk 1]
```diff
@@ -505,15 +505,17 @@ Phase.prototype.calculateStats = function() {
...
           var values = contextResults[key];
           var mean = stats.mean(values);
+          var stdev = stats.stdev(values);
 
           statistics[contextKey].push({
             Metric: key,
             Mean: mean,
             Median: stats.median(values),
             Min: Math.min.apply(Math, values),
+            '95th Perc.': stats.percentile(values, 0.95) || mean,
```
Comment (eliperelman) [inline lib/phases/phase.js:515]: How about making this just `95%`?

Comment (stasm) [inline lib/phases/phase.js:515]: Wouldn't that be easily confused with the 95% confidence interval? How about we just remove this altogether?

Comment (eliperelman) [inline lib/phases/phase.js:515]: The tough thing is the current dependency with how things are done right now, with the way things should actually be. For example, right now all of our metrics in the dashboards display p95. All of our baselines for v2.2 are calculated using p95. All our regression detection scripts query for p95.

All that is to say: we could easily switch everything over to use mean, but we would have to change the messaging around our releases to say that we are abandoning p95 for another alternative. Thoughts?

FYI, here is all the stats for v2.2 I gathered. https://gist.github.com/eliperelman/5aac3b94e2f815dbb827

Comment (eliperelman) [inline lib/phases/phase.js:515]: OK, remove p95, use 95% Bound instead.

Comment (eliperelman) [inline lib/phases/phase.js:515]: I'll merge this, remove p95, then publish as Raptor v3.0.

**Pull request merged**